src/test/java/teammates/test/cases/webapi/GetCourseActionTest.java

```diff
@@ -29,38 +30,61 @@ protected String getRequestMethod() {
     @Override
     protected void testExecute() throws Exception {
         InstructorAttributes instructor1OfCourse1 = typicalBundle.instructors.get("instructor1OfCourse1");
-        CourseAttributes courseAttributes = logic.getCourse(instructor1OfCourse1.getCourseId());
+        CourseAttributes expectedCourse = logic.getCourse(instructor1OfCourse1.getCourseId());
 
         loginAsInstructor(instructor1OfCourse1.googleId);
 
         ______TS("Not enough parameters");
 
         verifyHttpParameterFailure();
 
-        ______TS("typical success case");
+        ______TS("typical success case for instructor");
```

Comment:
Instead of using `____TS()`. It's better to separate tests into different sub tests and annotate them with `@Test`. Since you already did it for one of the function, let's do it for all of them?